src/Generating/Trees.cpp

```diff
@@ -29,6 +29,38 @@ static const sCoords Corners[] =
 	{1, 1},
 } ;
 
+// Array with possible directions for a branch to go to.
+static const Vector3d & pickBranchDirection(cNoise a_Noise, Vector3i a_BlockPos, int a_Seq)
```

Comment:
Not a big deal, but I think the signature should be:
```cpp
static const Vector3d pickBranchDirection(const cNoise & a_Noise, Vector3i a_BlockPos, int a_Seq)
```

Reply:
I'll move it into `cTree` while I'm at it.